src/version.php

```diff
@@ -3,5 +3,6 @@
     define('WOVN_PHP_NAME', 'WOVN.php');
 }
 if (!defined('WOVN_PHP_VERSION')) {
-    define('WOVN_PHP_VERSION', '0.1.23');
+    $version = isset($_ENV['WOVN_ENV']) && $_ENV['WOVN_ENV'] === 'development' ? 'VERSION' : '0.1.24';
```

Comment:
Just noticed that we should start increment minor version number (0.2.0) for these kinds of features.

Reply:
sorry, I didn't notice this.

Reply:
no problem, can do it next time

Reply:
How about increment to `1.0.0`.
Because this is not beta version.

Reply:
Yes, definitely a good idea